slack.json

```diff
@@ -3272,7 +3305,7 @@
                     "order": 1
                 },
                 "responses": {
-                    "description": "List of possible responses",
+                    "description": "List of possible responses (Maximum responses allowed are 5)",
```

Comment:
Seems like this could be a number instead

Reply:
Hello @alexa-phantom , 
The responses would be in comma separated string form only. On the App end the responses are split using commas and then stored in a list. While posting the question, the slack API allows at most 5 responses. Therefore we have added that in the description .